docs/sources/mimir/operators-guide/configure/about-versioning.md

```diff
@@ -51,6 +51,7 @@ The following features are currently experimental:
   - Disable alerting and recording rules evaluation on a per-tenant basis
     - `-ruler.recording-rules-evaluation-enabled`
     - `-ruler.alerting-rules-evaluation-enabled`
+  - Aligning of evaluation timestamp on interval ()
```

Comment:
```suggestion
  - Aligning of evaluation timestamp on interval (`align_evaluation_time_on_interval`)
```

Reply:
Fixed in https://github.com/grafana/mimir/pull/4014.